docker.go

```diff
@@ -57,6 +67,21 @@ func runDockerPin(cmd *cobra.Command, args []string) error {
 	return ioutil.WriteFile(ifDash(*dockerfile, "/dev/stdout"), n, 0644)
 }
 
+func runDockerBasePin(cmd *cobra.Command, args []string) error {
+	baseImageAndVersion := args[0]
+	baseImage := strings.Split(baseImageAndVersion, ":")[0]
+	dockerClient, err := docker.NewClientWithOpts(docker.FromEnv)
+	if err != nil {
+		return err
+	}
+	di, diErr := dockerClient.DistributionInspect(cmd.Context(), baseImageAndVersion, "")
+	if diErr != nil {
+		return diErr
+	}
+	hashedBase := baseImage + "@" + string(di.Descriptor.Digest) + "\n"
```

Comment:
Should we leave the tag included here? I suppose it's nice for readers to see what the image is based on, so they know how large the upgrade they're gonna do is.

Reply:
good point, of course the tag should stay